sample/lib/ui/pages/home_page.dart

```diff
@@ -1,13 +1,133 @@
+import 'package:GTUBT/models/post.dart';
 import 'package:flutter/material.dart';
 
 class HomePage extends StatelessWidget {
+  Post post1 = new Post(
+      title: "title1",
+      summary:
+          "text1 LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET",
+      coverImageUrl: "https://via.placeholder.com/350x120.png",
+      startDate: "05-05-2020");
+  Post post2 = new Post(
+      title: "title2",
+      summary:
+          "text2 LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET",
+      coverImageUrl: "https://via.placeholder.com/350x120.png",
+      startDate: "04-05-2020");
+  Post post3 = new Post(
+      title: "title3",
+      summary:
+          "text3 LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET",
+      coverImageUrl: "https://via.placeholder.com/350x120.png",
+      startDate: "04-05-2020");
+  Post post4 = new Post(
+      title: "title4",
+      summary:
+          "text4 LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET",
+      coverImageUrl: "https://via.placeholder.com/350x120.png",
+      startDate: "03-05-2020");
+  Post post5 = new Post(
+      title: "title5",
+      summary:
+          "text5 LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET",
+      coverImageUrl: "https://via.placeholder.com/350x120.png",
+      startDate: "02-05-2020");
```

Comment:
Burdaki date in formatindan emin miyiz? Modelleri date tipine cevirdikten sonra alalim bunu

Reply:
Benim lojikte iki tarihin günü aynı mı bakmam gerekiyor. Date objesi gelince bana startDate olarak gününü rahatça karşılaştırabilirim. Şimdilik böyle yaptım